### ISSUE TITLE
make all fails

### ISSUE THREAD

**Issue by TieSKey** [christian-roggia/vndb#2]
Not sure if it's just on my end though. Running docker latest version over Windows 10 ltsb.
Cloned the git repo from the first link (docker branch) and run the line end commands.

With only this message:

```
 ---> Running in dd62b4c69870
mkdir -p static/ch;
for i in $(seq -w 0 1 99); do mkdir -p "static/ch/$i"; done
mkdir -p static/f
mkdir -p static/cv;
for i in $(seq -w 0 1 99); do mkdir -p "static/cv/$i"; done
mkdir -p static/sf;
for i in $(seq -w 0 1 99); do mkdir -p "static/sf/$i"; done
mkdir -p static/st;
for i in $(seq -w 0 1 99); do mkdir -p "static/st/$i"; done
mkdir -p data/log
mkdir -p www
mkdir -p www/feeds
mkdir -p www/api
util/jsgen.pl
make: util/jsgen.pl: Command not found
make: *** [static/f/vndb.js] Error 127
Makefile:71: recipe for target 'static/f/vndb.js' failed
Service 'migration' failed to build: The command '/bin/sh -c make all' returned a non-zero code: 2
```

If I change the makefile code this way:
```
...
static/f/vndb.js: data/js/*.js util/jsgen.pl data/config.pl data/global.pl static/f
	chmod 777 util/jsgen.pl
	perl util/jsgen.pl
...

```	

Then I get the following error, seems to be a perl one this time:
```
 ---> Running in fe94e601c139
mkdir -p static/ch;
for i in $(seq -w 0 1 99); do mkdir -p "static/ch/$i"; done
mkdir -p static/f
mkdir -p static/cv;
for i in $(seq -w 0 1 99); do mkdir -p "static/cv/$i"; done
mkdir -p static/sf;
for i in $(seq -w 0 1 99); do mkdir -p "static/sf/$i"; done
mkdir -p static/st;
for i in $(seq -w 0 1 99); do mkdir -p "static/st/$i"; done
mkdir -p data/log
mkdir -p www
mkdir -p www/feeds
mkdir -p www/api
chmod 777 util/jsgen.pl
perl util/jsgen.pl .
No such file or directory at util/jsgen.pl line 59.
make: *** [static/f/vndb.js] Error 2
Makefile:71: recipe for target 'static/f/vndb.js' failed
Service 'migration' failed to build: The command '/bin/sh -c make all' returned a non-zero code: 2
```

**Comment by christian-roggia** [comment]
Thank you for reporting this issue, it seems you are still having problems with the LF/CRLF line endings conflict.

Please clone the repository again with the following command:
> git clone https://github.com/christian-roggia/vndb.git --config core.eol=lf --config core.autocrlf=input

I'll update the README and make it more clear for Windows users.

**Comment by christian-roggia** [comment]
Fixed in 355cedec5551d3433429a95a9e54521fe5e9afb1.
